pkg/operator/defragcontroller/defragcontroller.go

```diff
@@ -46,7 +49,7 @@ func NewDefragController(
 		etcdClient:           etcdClient,
 		infrastructureLister: infrastructureLister,
 	}
-	return factory.New().ResyncEvery(9*time.Minute).WithInformers(
+	return factory.New().ResyncEvery(11*time.Minute).WithInformers( // compaction is every 10m
```

Comment:
Nit: wonder if we could for clarity have a constant called something like: `compactionTime` and here do `compactionTime + 1*time.Minute`.